### PR TITLE
add eink-light theme

### PR DESCRIPTION
**Pull request by Tsyron**
I made an e-ink theme

### CODE REVIEW

File: themes/eink-light/v1/eink-light
Comment (thecodrr) [inline themes/eink-light/v1/eink-light:1]: @Tsyron you need to give the theme `.json` extension.

**Review by thecodrr — CHANGES_REQUESTED** [review]

File: themes/eink-light/v1/eink-light
Comment (Tsyron) [reply themes/eink-light/v1/eink-light:1]: oh sorry, I didn't see that. I've changed the name

File: themes/eink-light/v1/eink-light.json
Comment (thecodrr) [inline themes/eink-light/v1/eink-light.json:1]: Forgot to mention, the theme file must be named `theme.json`.

**Review by thecodrr — CHANGES_REQUESTED** [review]

**Comment by Tsyron** [comment]
yes, this is true, but text that's slightly grey is hard to read on e-ink

**Comment by thecodrr** [comment]
What do you think @Starship-Headlight? Can we make the text black in your `LIGHT & DARK` theme? Both of these themes are identical aside from the text color.

**Pull request closed without merging**